packages/presentation/lib/screens/login/data/login_data.dart

```diff
@@ -0,0 +1,26 @@
+class LoginData {
+  final String _login;
+  final String _password;
+
+  LoginData(
+    this._login,
+    this._password,
+  );
+
+  String get login => _login;
+  String get password => _password;
```

Comment:
seems to be redundant. why not to make _login public? it's final, so, no value van be set to it?

Reply:
Fixed.